notebook/agentchat_mongodb_RetrieveChat.ipynb

```diff
@@ -0,0 +1,269 @@
+    "# This only applies to files under the directories in `docs_path`. Explicitly included files and urls will be chunked regardless of their types.\n",
+    "# In this example, we set it to [\"non-existent-type\"] to only process markdown files. Since no \"non-existent-type\" files are included in the `websit/docs`,\n",
+    "# no files there will be processed. However, the explicitly included urls will still be processed.\n",
+    "ragproxyagent = RetrieveUserProxyAgent(\n",
+    "    name=\"ragproxyagent\",\n",
+    "    human_input_mode=\"NEVER\",\n",
+    "    max_consecutive_auto_reply=3,\n",
+    "    retrieve_config={\n",
+    "        \"task\": \"code\",\n",
+    "        \"docs_path\": [\n",
+    "            \"https://raw.githubusercontent.com/microsoft/FLAML/main/website/docs/Examples/Integrate%20-%20Spark.md\",\n",
+    "            \"https://raw.githubusercontent.com/microsoft/FLAML/main/website/docs/Research.md\",\n",
+    "            os.path.join(os.path.abspath(\"\"), \"..\", \"website\", \"docs\"),\n",
+    "        ],\n",
+    "        \"custom_text_types\": [\"non-existent-type\"],\n",
+    "        \"chunk_token_size\": 2000,\n",
+    "        \"model\": config_list[0][\"model\"],\n",
+    "        \"vector_db\": \"mongodb\",  # MongoDB Atlas database\n",
+    "        \"collection_name\": \"flaml_collection_two\",\n",
+    "        \"index_name\": \"flaml_index_two\",\n",
+    "        \"db_config\": {\n",
+    "            \"connection_string\": \"<connection_string_goes_here>\",  # MongoDB Atlas connection string\n",
+    "            \"database_name\": \"test_db\",  # MongoDB Atlas database\n",
+    "        },\n",
+    "        \"get_or_create\": False,  # set to False if you don't want to reuse an existing collection\n",
+    "        \"overwrite\": True,  # set to True if you want to overwrite an existing collection\n",
+    "    },\n",
+    "    code_execution_config=False,  # set to False if you don't want to execute the code\n",
+    ")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Example 1\n",
+    "\n",
+    "[Back to top](#table-of-contents)\n",
+    "\n",
+    "Use RetrieveChat to help generate sample code and automatically run the code and fix errors if there is any.\n",
+    "\n",
+    "Problem: Which API should I use if I want to use FLAML for a classification task and I want to train the model in 30 seconds. Use spark to parallel the training. Force cancel jobs if time limit is reached."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "VectorDB returns doc_ids:  [[]]\n",
+      "\u001b[32mNo more context, will terminate.\u001b[0m\n",
+      "\u001b[33mragproxyagent\u001b[0m (to assistant):\n",
+      "\n",
+      "TERMINATE\n",
```

Comment:
The retrieve_docs is not working as expected. No doc is returned. Either the query pipeline or the atlas local env is not functional.

Reply:
On it! Thank you so much for helping debug! There is something wrong with the implementation I believe -- will debug this shortly.

Reply:
@thinkall - It is an implementation bug! its an issue with the index_name :) WIl fix shortly.